src/hotspot/os/windows/globals_windows.hpp

```diff
@@ -39,6 +39,10 @@
 product(bool, UseAllWindowsProcessorGroups, false,                        \
         "Use all processor groups on supported Windows versions")         \
                                                                           \
+product(bool, EnableAllLargePageSizes, false, EXPERIMENTAL,               \
```

Comment:
NIT: I think this may need to be more specific, otherwise it may confuse people thinking that it applies to other OS'es as well.